pom.xml

```diff
@@ -22,6 +22,9 @@
       <resource>
         <directory>src/main/site</directory>
       </resource>
+      <resource>
```

Comment:
Eclipses uses this, and configures it as a resource directory, which makes it easier to navigate.